### ISSUE TITLE
effet visuel OBLIGATOIRE ;)

### ISSUE THREAD

**Issue by farfadet46** [ElementW/Diggler#1]
si tu peut c'est un must-have ;)
Capture issue de http://galaxy55.com/ (jeu abandonné malheureusement)
![galaxy-55](https://cloud.githubusercontent.com/assets/11613522/12217091/c496dcde-b6f5-11e5-845b-bd0d3d5e7c03.png)

Autre exemple ; un shader sous UNITY3D :
curved world: https://www.assetstore.unity3d.com/en/#!/content/26165

**Comment by ElementW** [comment]
:astonished: Not a bad idea at all... I take note. Expect it... some time in the future :)
P.S.: eww, Unity3D. Too bad the shader has to be bought, but I know how to implement that already.

---

:astonished: Pas mal du tout... Je pend note. Attend-toi à ça... à un moment dans le futur :)
P.S.: berk, Unity3D. Dommage que le shader soit payant, mais je vois déjà comment implémenter ça.

**Comment by farfadet46** [comment]
yep, c'est cher j'ai déjà acheté Playmaker (genre de programmation visuelle pour unity) à 30€ environs. mais là c'est juste un effet visuel qui ne serait utilisé qu'une fois du coup NOPE pas de sous pour ça.
